code/game/machinery/kitchen/cooking_machines/_appliance.dm

```diff
@@ -110,14 +110,18 @@
 		icon_state = off_icon
 
 /obj/machinery/appliance/verb/toggle_power()
-	set src in view()
 	set name = "Toggle Power"
 	set category = "Object"
+	set src in world //So that AI can operate it remotely
```

Comment:
Same here. Add it to the right click menu but remove it from the object tab.
"Toggle Power" in the object tab of an AI can mean anything.